libnmstate/schema.py

```diff
@@ -210,3 +212,80 @@ class OVSBridge(object):
 class OVSBridgePortType(object):
     SYSTEM = 'system'
     INTERNAL = 'internal'
+
+
+class Team(object):
+    TYPE = InterfaceType.TEAM
+    CONFIG_SUBTREE = 'team'
+
+    SLAVES = 'slaves'
+    CONFIG = 'config'
+
+
+class TeamConfig(object):
+    CONFIG = Team.CONFIG
+
+    DEVICE = 'device'
+    DEBUG_LEVEL = 'debug_level'
+    HWADDR = 'hwaddr'
+
+    RUNNER = 'runner'
```

Comment:
That is a lot of parameters.

@EdDev what do you usually do in nmstate, do you keep minimal useful subset or is it fine to have all those?

Reply:
Yes it is. If we split them into subsets it will result in a lot of classes. Teamd JSON API have a lot of fields and possible values.

See also: man 5 teamd.conf

Reply:
You may use the way we did for bond options.

This is not a good practice to include every options in schema, just allow arbitrary dictionary and depend on backend(NM) to check it.

Reply:
Why is it not a good practice? It seems to be more user-friendly to fail as quick as possibly if there is an error and the first checks are the schema validation.

@EdDev what is your opinion on this?

Reply:
Whenever team changed their option or add more, we need to sync with that list.

If team decide to break back compatibility, with that hard coded in schema, we also need to
break our back compatibility.

Anyway, we don't need to discuss on this as this PR should only support `slaves` option in
the team interface.